### ISSUE TITLE
Maven (maybe)

### ISSUE THREAD

**Issue by Hockeyfan360** [Superior-Development/IcarusMod#163]
Do you think that it would be a good idea to move to a maven based project?

**Comment by Wild1145** [comment]
Not really... We do only have one dependency... Don't really see any value to it to be honest

**Comment by CameronRedmore** [comment]
Hmm, I agree with Wild, whilst this would simplify and expand our library availability I'm not entirely certain it is required. The current ANT build scheme works well enough for our purposes.
